test/test_cpp_extensions_mtia_backend.py

```diff
@@ -34,7 +35,7 @@ def remove_build_path():
 
 
 @unittest.skipIf(
-    IS_ARM64 or not IS_LINUX or TEST_CUDA or TEST_PRIVATEUSE1 or TEST_ROCM,
+    IS_ARM64 or not IS_LINUX or TEST_CUDA or TEST_PRIVATEUSE1 or TEST_ROCM or TEST_XPU,
```

Comment:
It is mutually exclusive to xpu.

Reply:
cc @egienvalue with me want to add a TEST_MTIA to common_utils above to simplify this kind of statement!

Reply:
I see. Thanks for the suggestion. I will do that.